.github/workflows/integration_testing.yml

```diff
@@ -73,6 +79,35 @@ jobs:
           disableRateLimiting: true
           customModules: synapse_super_invites
           customConfig: |
+            email:
+              smtp_host: localhost
+              smtp_port: 1025
+              smtp_user: "test1"
+              smtp_pass: "test"
+              force_tls: false
+              require_transport_security: false
+              enable_tls: false
+              notif_from: "Your Friendly %(app)s homeserver <noreply@example.org>"
+              app_name: Acter
+              enable_notifs: true
+              notif_for_new_users: false
+              client_base_url: "http://localhost/riot"
+              validation_token_lifetime: 15m
+              invite_client_location: https://app.element.io
+              can_verify_email: true
+
+              subjects:
+                message_from_person_in_room: "[%(app)s] You have a message on %(app)s from %(person)s in the %(room)s room..."
+                message_from_person: "[%(app)s] You have a message on %(app)s from %(person)s..."
+                messages_from_person: "[%(app)s] You have messages on %(app)s from %(person)s..."
+                messages_in_room: "[%(app)s] You have messages on %(app)s in the %(room)s room..."
+                messages_in_room_and_others: "[%(app)s] You have messages on %(app)s in the %(room)s room and others..."
+                messages_from_person_and_others: "[%(app)s] You have messages on %(app)s from %(person)s and others..."
+                invite_from_person_to_room: "[%(app)s] %(person)s has invited you to join the %(room)s room on %(app)s..."
+                invite_from_person: "[%(app)s] %(person)s has invited you to chat on %(app)s..."
+                password_reset: "[%(server_name)s] Password reset"
+                email_validation: "[%(server_name)s] Validate your email"
```

Comment:
not sure these are necesary ... are we sure these are needed? I'd like to keep configuration changes in the tests at a minimum.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1783/commits/2e18a453ad8f45c0894307f47f2fd03a655a7283.
Left only required items.